autoload/picker.vim

```diff
@@ -271,31 +271,41 @@ function! s:PickFile(list_command, vim_command) abort
     call s:Picker(a:list_command, a:vim_command, l:callback)
 endfunction
 
-function! picker#CheckIsNumber(variable, name) abort
-    " Print an error message if variable is not of type Number.
+function! picker#IsNumber(variable) abort
+    " Determine if a variable is a number.
     "
     " Parameters
     " ----------
     " variable : Any
     "     Value of the variable.
-    " name : String
-    "     Name of the variable.
-    if type(a:variable) != type(0)
-        echoerr 'vim-picker:' a:name 'must be a number'
+    "
+    " Returns
+    " -------
+    " Boolean
+    "     v:true if the variable is a number, v:false otherwise.
+    if type(a:variable) ==# type(0)
+        return v:true
+    else
+        return v:false
```

Comment:
Maybe drop the `if` statements and just `return type(a:variable) ==# type(0)`?

Vim/Neovim doesn't have a type devoted to boolean.  `:help v:true` suggests the constant is there to support JSON encoding.  The commented boolean return value still applies even if the function returns 0/1 for false/true.